python/res/enkf/enkf_main.py

```diff
@@ -99,6 +98,11 @@ def umount(self):
         if self.__fs_manager is not None:
             self.__fs_manager.umount()
 
+    def runWorkflows(self , runtime):
```

Comment:
This is not really a fix is it? It just reintroduces the cyclic dependency..

Reply:
Right, there is probably a better solution to this, could be to do: 
```

    def runWorkflows(self , runtime):
        """:type res.enkf.enum.HookRuntimeEnum"""
        hook_manager = self._enkf_main().getHookManager()
        hook_manager.runWorkflows(runtime  , self._enkf_main())
```
to:
```
    @staticmethod
    def runWorkflows(ert , runtime):
        """:type res.enkf.enum.HookRuntimeEnum"""
        hook_manager = ert._enkf_main().getHookManager()
        hook_manager.runWorkflows(runtime  , ert)
```
Should probably have a discussion on how we can solve this, is required to run most internal python workflows.

Reply:
This has been implemented in #744